sktime/transformations/panel/catch22.py

```diff
@@ -603,4 +498,7 @@ def get_test_params(cls, parameter_set="default"):
         """
         param1 = {}
         param2 = {"features": "DN_HistogramMode_5"}
-        return [param1, param2]
+        param3 = {"features": [1, 12, 21]}
+        param4 = {"features": ["forecast_error", "centroid_freq"]}
+        param5 = {"features": [11, "DN_HistogramMode_5", "forecast_error"]}
```

Comment:
here we should cover the other parameters too. There are a couple. `n_jobs` is not needed, ofc.

I think we can get good coverage with 5 sets, just add one version each, so each value is used somewhere, no need to multiply exponentially.